var/spack/repos/builtin/packages/freecad/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Freecad(CMakePackage):
+    """FreeCAD is an open-source parametric 3D modeler made primarily
+    to design real-life objects of any size. Parametric modeling
+    allows you to easily modify your design by going back into your
+    model history to change its parameters."""
+
+    homepage = "https://www.freecad.org/"
+    url = "https://github.com/FreeCAD/FreeCAD/archive/refs/tags/0.20.2.tar.gz"
+
+    maintainers("aweits")
+
+    version("0.20.2", sha256="46922f3a477e742e1a89cd5346692d63aebb2b67af887b3e463e094a4ae055da")
+
+    depends_on("opencascade")
+    depends_on("xerces-c")
+    depends_on("vtk")
+    depends_on("salome-med")
+    depends_on(
+        "boost+python+filesystem+date_time+graph+iostreams+program_options+regex+serialization+system+thread"  # noqa: E501
+    )
+    depends_on("qt@5:")
```

Comment:
Any idea if it works with the new qt-base? Might make the build easier

Reply:
does not work